test/smoke_test_facebook-bart-large-cnn.py

```diff
@@ -53,6 +54,7 @@ def testSum(self):
         self.assertEqual(req_body['text'], res_body['text'])
         self.assertTrue(str(res_body['summary']).find("Weaviate"))
         self.assertGreater(len(str(res_body['summary']).strip()), 0)
+        print(f"Test 2 result: {str(res_body['summary'][0]['result'])}")
```

Comment:
remove prints again?

Reply:
I will remove those prints in the next PR